### ISSUE TITLE
Nokaled BUG on Enhance shaman?

### ISSUE THREAD

**Issue by twohead2** [gamefreedomgit/Maelstrom#5785]
(https://discord.com/channels/1040715551531618365/1058491933015691424/1217965754889011210)  Notice the Spec change and the proc damage changes

NOKALED HC is bugged on Enhance shaman

IF you see with resto or ele spec, Inspect Nokaled Normal and HC, the procs are working fine, But when you change Specs to Enhance, the Normal Nokaled has better proc damage(as written in the Item description) and the HC one has less. Is this because the scaling on Nokaled HC is bugged?

**How it should work:** The Nokaled HC SHOULD have better proc dmg than the normal since getting it would just mean like 30 more agi for less proc dmg.

**Database links:**

**Comment by bonfax4** [comment]
Why not test and see if it's only a tooltip issue ?

**Comment by twohead2** [comment]
> Why not test and see if it's only a tooltip issue ?

I tested it and it does virtually the same damage on iceblast and flameblast, and for shadowblast the HC one is better, is this supposed to be how it is? The heroic one should proc for more damage and it does the same amount 14k-17k on the procs excluding shadowblast.

**Comment by bonfax4** [comment]
Sounds like the HC is not being affected by mastery then.

**Comment by bonfax4** [comment]
But post logs to confirm. I'm reading the HC and lfr version display the shadow portion on their tooltip, so you're see lower dmg. Normal one displays the fire proc which is affected by mastery

**Comment by twohead2** [comment]
> But post logs to confirm. I'm reading the HC and lfr version display the shadow portion on their tooltip, so you'd see lower dmg on the tooltip only. Normal one displays the fire proc which is affected by mastery
> 
> Oh and the dmg range does interlap a little bit, so you'll need a few hits in to make sure.
> 
> Normal : 7654-11481 HC : 8640 -12960

yeah I tested it a lot but since it has a very low proc chance unless you get really lucky, its hard to know which one is better, i guess it adds up that sometimes i got consecutive low rolls on the HC one and high rolls on the normal, in my experience, the normal one has always CRIT harder but the max proc is dominated by the HC one for normal hits. I guess I overestimated the difference between the normal and HC procs as i believed they would be MUCH better and also since the tooltip was correct on my resto spec but not on enhance, it led to further confusion. It turns out that on average the HC one is about 1-1.5k dps boost specifically on the procs. Thank you for the time and input.
